Specs/GCMV8.lean

```diff
@@ -0,0 +1,262 @@
+/-
+Copyright (c) 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Author(s): Yan Peng
+-/
+import Arm.BitVec
+import Specs.GCM
+
+-- References:
+-- https://github.com/GaloisInc/cryptol/blob/master/lib/Cryptol/Reference.cry
+-- https://tiny.amazon.com/12296d03c/githGalocrypblob1e31PrimSymm
+-- https://tiny.amazon.com/3edw4edm/githawslawslblob02aaSAWspec
+-- https://github.com/awslabs/aws-lc-verification/blob/master/SAW/proof/AES/GHASH.saw
+-- https://link.springer.com/chapter/10.1007/978-3-031-34671-2_30
+-- https://www.rfc-editor.org/rfc/rfc8452.txt
+
+namespace GCMV8
+
+open BitVec
+
+------------------------------------------------------------------------------
+-- Functions related to bit vectors
+
+/-- Reverse bits of a bit-vector. -/
+def reverse (x : BitVec n) : BitVec n :=
```

Comment:
I'd recommend moving this to `Arm/BitVec.lean` in the `BitVec` namespace.

Reply:
Done.